web/src/components/ExpandedViewMode/ExpandedViewMode.js

```diff
@@ -18,6 +18,8 @@ import RightMenu from './RightMenu/RightMenu'
 import ExpandedViewModeHeader from './ExpandedViewModeHeader/ExpandedViewModeHeader'
 import ExpandedViewModeContent from './ExpandedViewModeContent/ExpandedViewModeContent'
 import ExpandedViewModeFooter from './ExpandedViewModeFooter/ExpandedViewModeFooter'
+import { startTitleEdit, endTitleEdit, startDescriptionEdit, endDescriptionEdit } from '../../goal-editing/actions'
+import { sendRealtimeInfoSignal } from '../../realtime-info-signal/actions'
```

Comment:
I guess you could remove this from here now
```suggestion
```

plus corresponding changes to this file